common/api/core-frontend.api.md

```diff
@@ -14018,10 +14010,10 @@ export abstract class Viewport implements IDisposable, TileUser {
     getFrustum(sys?: CoordSystem, adjustedBox?: boolean, box?: Frustum): Frustum;
     // @alpha (undocumented)
     getMapFeatureInfo(hit: HitDetail): Promise<MapFeatureInfo>;
-    // @internal (undocumented)
     getMapLayerImageryProvider(index: number, isOverlay: boolean): MapLayerImageryProvider | undefined;
     // @internal
     getMapLayerIndexesFromIds(mapTreeId: Id64String, layerTreeId: Id64String): MapLayerIndex[];
+    getMapLayerRange(layerIndex: number, isOverlay: boolean): Promise<MapCartoRectangle | undefined>;

```

Comment:
Why don't all these function that take `layerIndex` and `isOverlay` take `MapLayerIndex` instead (besides it being `internal`)?

Reply:
I don't see any reason they couldn't take that instead. Although many of the functions would still separate out `layerIndex` and `isOverlay` anyway, so wouldn't be simplified much aside from the parameters. @mdastous-bentley do you think this is worth changing/were you guys planning to use `MapLayerIndex` outside of `MapTiledGraphicsProvider`?

Reply:
Just made a commit changing this for most functions that take `isOverlay` and `layerIndex`, excluding just [moveMapLayerToIndex ](https://github.com/iTwin/itwinjs-core/blob/eringram/maplayers-remove-internals/core/frontend/src/DisplayStyleState.ts#L596)which takes 2 indices.